pkg/vetter/conflictingvirtualservicehost/vet_test.go

```diff
@@ -462,6 +475,53 @@ var _ = Describe("Conflicting Virtual Service Host Vet Notes", func() {
 			Expect(vsNotes).To(BeEmpty())
 		})
 
+
+		// Conflict in same VS.
+		It("Warn if two routes conflict in the same VS", func() {
```

Comment:
I'd update this to something like:
"Generates a note for each host if a VS with multiple hosts has conflicts"

I was genuinely confused why this generated more than one note until I saw that the hosts were different and went back to the definition of Vs1 and saw that it had multiple hosts.